src/main/java/pack/service/impl/PersonService.java

```
@@ -0,0 +1,59 @@
+package pack.service.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.annotation.PostConstruct;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import pack.model.Person;
+import pack.repository.IPersonRepository;
+import pack.service.IPersonService;
+
```

For all the service methods, if something goes wrong, please throw a custom exception,